test/unit/processing/glos/group-by-invoice-number.test.js

```diff
@@ -0,0 +1,95 @@
+const mockCorrelationId = require('../../../mocks/correlation-id')
+const { filename1 } = require('../../../mocks/glos-filenames')
+
+const groupByInvoiceNumber = require('../../../../app/processing/glos/group-by-invoice-number')
+
+describe('group by invoice number', () => {
+  test('should group by invoice number with 2 invoice lines', () => {
+    const paymentRequests = [{
```

Comment:
Could we define the paymentyRequest once and then reuse it with the spread operator and then only have to change the invoice number. 

This would tidy up the file a bit.

Example:
```
const glosPaymentRequest = {
     correlationId: mockCorrelationId,
      batch: filename1,
      batchExportDate: '31/05/2023 22:01:38',
      invoiceNumber: '33315 16',
      paymentRequestNumber: 1,
      frn: '1102294241',
      sbi: '106609512',
      claimDate: '31/05/2023 22:01:38',
      standardCode: '028Z141Q',
      description: 'English Woodland Grant Scheme',
      value: 81
}
```

Then in each test
 
```
const paymentRequests = [{
      ...glosPaymentRequest,
      invoiceNumber: '33315 16'
    }, {
      ...glosPaymentRequest,
      invoiceNumber: '33315 16'
    }]
```

Reply:
Oh okay yeah, I forgot to add that. Will do so now thanks :)